admin/howto/control-user-server.md

```diff
@@ -0,0 +1,48 @@
+# Controlling a user's server
+
+Hub admins can unilaterally perform actions on user's servers via the
+**Administrator's Panel**. These are primarily used to debug user's session
+easily.
+
+You can access the admin panel by clicking the 'Admin' button in the top bar
+in your hub control panel.  Alternatively, you can go to this URL in our
+browser: `https://<your-hub-url>/hub/admin`.
+
+
+## Access a user's server
+
+Accessing a user's server is very useful when trying to debug or reproduce an issue they might have. This facility is available to admins via the admin panel. 
```

Comment:
I wonder if we should have a section on "Admin Panel" that just shows people how to get there, and then when we use `Admin Panel` in other docs, it can link to that section

Reply:
I think for now, I wanna just repeat them. We can link out later - how does that sound?